src/main/resources/public/js/ts_scripts.js

```diff
@@ -1579,7 +1579,7 @@ $(document).ready(function () {
                 "thisChipSet": 'chipSetBGW' //YANG $("#chipSetList .active").attr("id")
             };
 
-            if ((chipstripwindow != null) || (chipstripwindow.closed == false)) {      //if the window is open then send message to change the chip set
+            if ((chipstripwindow != null) || (chipstripwindow.closed === false)) {      //if the window is open then send message to change the chip set
```

Comment:
chipstripwindow !== null

Reply:
This logic also does not make sense.  If chipstripwindow is null, it will try to check for chipstripwindow.closed and cause an error because there is no closed property of null.  Do you mean `if ((chipstripwindow !== null) && (chipstripwindow.closed === false))`?